test/runtests.jl

```diff
@@ -126,6 +126,13 @@ end
         @test Gray24(0.8)*0.5 === Gray(0.4)
         @test Gray24(0.8)/2   === Gray(0.5f0*N0f8(0.8))
         @test Gray24(0.8)/2.0 === Gray(0.4)
+
+        # Complement
+        @test complement(Gray(0.5)) == Gray(0.5)
+        @test complement(Gray(0.2)) == Gray(0.8)
+        @test all(complement.(img) .== 1 .- img)
+        # deprecated (#690)
```

Comment:
the link is invalid now, needs update to https://github.com/JuliaImages/Images.jl/issues/690